test/Command.js

```diff
@@ -187,7 +184,75 @@ describe("Command", () => {
   });
 
   describe(".runPreparations()", () => {
-    it("needs tests");
+    describe(".filteredPackages", () => {
+      let testDir;
+
+      beforeEach(() => initFixture("UpdatedCommand/basic").then((dir) => {
+        testDir = dir;
+      }));
+
+      class TestCommand extends Command {}
+
+      function cli(cmd, ...args) {
+        return execa.sync(cmd, args, { cwd: testDir }).stdout.toString("utf-8");
+      }
+
+      function run(opts) {
+        const cmd = new TestCommand([], opts, testDir);
+        cmd.run();
+        return cmd;
+      }
+
+      it("--scope should filter packages", () => {
```

Comment:
This isn't part of the functionality that I'm adding, but I wanted to make sure this behaviour was tested in this manner, especially since I'm testing the combination of `--scope` and `--scope-updated`.